docs/toc.yml

```diff
@@ -1607,8 +1607,6 @@ items:
                     href: guide/azure-resource-manager/advanced-templates/index.md
                   - name: Update a resource
                     href: guide/azure-resource-manager/advanced-templates/update-resource.md
-                  - name: Conditionally deploy a resource
-                    href: guide/azure-resource-manager/advanced-templates/conditional-deploy.md
```

Comment:
We should be deleting the `conditional-deploy.md` file and adding a redirect as well as part of this PR.